.meteor/versions

```diff
@@ -120,6 +120,7 @@ mongo-id@1.0.6
 mongo-livedata@1.0.12
 mrt:later@1.6.1
 natestrauser:select2@4.0.3
+nimble:restivus@0.8.12
```

Comment:
This is equally valuable for making end points restful, however, it has no relationship with payment method and thus is an unnecessary file change

Reply:
Oh, that is true. I am working on it